src/server.h

```diff
@@ -1992,12 +1973,11 @@ struct redisServer {
     size_t blocking_op_nesting; /* Nesting level of blocking operation, used to reset blocked_last_cron. */
     long long blocked_last_cron; /* Indicate the mstime of the last time we did cron jobs from a blocking operation */
     /* Pubsub */
-    dict *pubsub_channels;  /* Map channels to list of subscribed clients */
+    kvstore *pubsub_channels;  /* Map channels to list of subscribed clients */
```

Comment:
please mention in the top comment, that you changed pubsub_channels to kvstore and why.

please also mention the reason in the code (e.g. here, and maybe somewhere in pubsub.c)

Reply:
done, but couldn't find an appropriate place to mention it in pubsub.c